README.md

```diff
@@ -18,6 +18,16 @@ time, as well as processing recorded output data. Both C++ and Python are suppor
     * [Running Examples](#running-examples)
 * [Usage](#usage)
 
+### Requirements
+
+#### C++ Support
+- C++11 or later
+- CMake 3.x or Bazel 3.x
+- GCC, Clang, or MSVC
```

Comment:
We should mark MSVC as preliminary until it is run by CI

Reply:
Agreed. Done.